docs/reference/vllm.md

```diff
@@ -47,6 +48,16 @@ curl http://127.0.0.1:8000/generate \
         }'
 ```
 
+To generate a string that matches the grammar `<grammar>`:
+
+```bash
+curl http://127.0.0.1:8000/generate \
+    -d '{
+        "prompt": "What is Pi? Give me the first 15 digits: ",
+        "cfg": <grammar>
```

Comment:
Would it be possible to add a (simple) example that users can copy/paste?

Reply:
fixed